keras/legacy/interfaces.py

```diff
@@ -74,3 +74,44 @@ def wrapper(*args, **kwargs):
 
         return func(*args, **kwargs)
     return wrapper
+
+
+def legacy_dropout_support(func):
+    """Function wrapper to convert the `dropout` constructor from Keras 1 to 2.
+
+    # Arguments
+        func: `__init__` method of `Dropout`.
+
+    # Returns
+        A constructor conversion wrapper.
+    """
+    @six.wraps(func)
+    def wrapper(*args, **kwargs):
+        if len(args) > 2:
+            # The first entry in `args` is `self`.
+            raise TypeError('The `Dropout` layer can have at most '
+                            'one positional argument (the `rate` argument).')
+
+        # p
```

Comment:
@fchollet  Thanks, 
 This part, as you show like Dense layer, 'Dropout' layer can also have at mosts one positional argument. That's why I keep this line.